tests/backend/Shared/Timebox.Shared.Events.Tests/ModuleEventPublisherTests.cs

```diff
@@ -54,14 +54,14 @@ public void PublishDomainEventAsync_Always_LogsAndReturnsIfNoSubscribers()
             //Arrange
             var sut = CreateSut();
             var module = "test";
-            _mocker.GetMock<IMessageKeyService>().Setup(o => o.GetKeyForMessage<TestMessageType>()).Returns(module);
+            _mocker.GetMock<IModuleOwnerKeyService>().Setup(o => o.GetKeyForMessage<TestMessageType>()).Returns(module);
 
             //Act
             sut.PublishDomainEventAsync(new TestMessageType());
             //Assert    
```

Comment:
Add assert to see if logger is called.

Reply:
Very difficult to asset the logger as moq does not support extension methods but may be worth looking into we can always wrap it I suppose what do you think ?

Reply:
It might be nice to wrap it and then it'll be extensible. I haven't used the logger, so I'll leave that decision to you 😉

Reply:
hahahahaha thanks rob